src/data/support-title.csv

```diff
@@ -67,5 +67,8 @@
 "[召しませふぁーすとBite！]","[Try First with Bite!]"
 "[おもい、ねがう]","[A Feeling, a Wish]"
 "[Head-on fight！]","[Head-on fight!]"
+"[明日は全国的に赤でしょう♪]","[Crimson to the Entire Country Tomorrow♪]"
+"[今宵、円舞曲にのせて]","[Tonight, Let's Dance to the Waltz]"
+"[ウマ王伝説・最強になった件]","[Legend of the Uma King・The Strongest]"
```

Comment:
The first part here is fine, but the 2nd completely leaves out most of the content. It reads more like "How [he] became the Strongest". That's too long though, so we'd need to reword it to be better, while still including relevant content.

Reply:
i really cant think of the proper words for the second part, so if anyone has a better one please let me know

Reply:
Well it does sound like a name for the first entry in a line of books, but I don't think we need to go for word-for-word, since it's a one-time title name.

I do agree that it sounds a bit stiff tho, something like "The Legend of the Strongest Uma King" would probably roll off the tongue better.

Reply:
Staying literal probably won't cut it yea, what you suggested sounds better to me already.

Reply:
Or, if you really wanted to include the second part and keep it really flowery, "The Legend of the Strongest Uma King and How He Came to Be", but that's getting a bit long lol

Reply:
That's definitely not doable for the patch i think lol

Reply:
yeah i think ill go with what gertasik suggests here, thanks!